src/docs/src/components/sidebar.tsx

```diff
@@ -26,25 +39,25 @@ const Sidebar: FunctionComponent<SidebarTypes> = ({ siteTitle }: SidebarTypes) =
     <div className="sidebar" role="navigation" aria-label="sidebar">
       <div>
         <h1 className="logo">
-          <Link
-            to="/"
-            style={{
-              textDecoration: 'none',
-            }}
-          >
+          <Link to="/" css={logoStyles}>
+            <span role="img" aria-label="logo">
+              🧑‍🎤
+            </span>
             {siteTitle}
           </Link>
         </h1>
       </div>
       <nav>
         <ul>
-          {data.allMdx.edges.map((edge) => (
-            <li key={edge.node.id}>
-              <Link to={edge.node.frontmatter.path}>
-                {edge.node.frontmatter.title}
-              </Link>
-            </li>
-          ))}
+          {data.allMdx.edges
+            .filter((edge) => edge.node.frontmatter.path)
```

Comment:
It will be better to have some type for edges

Reply:
Added